tools/flang2/utils/ilitp/CMakeLists.txt

```diff
@@ -13,8 +13,7 @@ target_compile_definitions(ilitp
   PRIVATE
   ILITP_UTIL
   PGF90=1
-  TARGET_X86
-  TARGET_X8664
+  TARGET_64
```

Comment:
Can we just delete `TARGET_X86` and `TARGET_X8664`? They aren't used under this directory at all.

Reply:
I've noticed that downstream we removed all of those TARGET_64 and TARGET_32 and no harm was done, so I'm reworking this old patch again.